releases/unreleased/support-authenticated-queries.yml

```diff
@@ -0,0 +1,14 @@
+---
+title: Support authenticated queries
+category: added
+author: Miguel Ángel Fernández <mafesan@bitergia.com>
+issue: 35
+notes: >
+    After this change, users can perform authenticated
+    queries to Bestiary. For doing so, they must
+    generate a JWT (JSON Web Token) which has to be
+    included within the `Authorization` header in the
+    HTTP requests. 
+
+    This change includes methods for generating,
+    verifying and refreshing these tokens.
```

Comment:
Don't talk about `this change`. You also should explain which methods were added and how to use them.